app/bolt_listeners.py

```diff
@@ -228,12 +240,16 @@ def respond_in_dm(
     client: WebClient,
     logger: logging.Logger,
 ):
+    logger.debug('respond to dm')
+    wip_reply = None
     try:
         openai_api_key = context.get("OPENAI_API_KEY")
 
+        correct_ts = 'thread_ts' if 'thread_ts' in payload else 'ts'
+
         messages_in_context = client.conversations_replies(
                 channel=context.channel_id,
-                ts=payload.get('thread_ts'),
+                ts=payload[correct_ts],
```

Comment:
Apparently, trying to "hack" the `thread_ts` didn't work and I needed to use the correct timestamp